src/GitLink/Extensions/ProjectExtensions.cs

```diff
@@ -94,10 +76,9 @@ public static string GetOutputPdbFile(this Project project)
         {
             Argument.IsNotNull(() => project);
 
-            var outputFile = project.GetOutputFile();
-            var pdbFile = Path.ChangeExtension(outputFile, ".pdb");
+            var outputFile = project.GetPropertyValue("PdbFile");
```

Comment:
Is the PdbFile property available everywhere? Just need to be careful that we are not breaking functionality for the people using VS 2013 / VS 2015 (although personally I only care about 2015 / 2017).